content/kubermatic/main/tutorials-howtos/monitoring-logging-alerting/master-seed/customization/thanos.md

```diff
@@ -0,0 +1,302 @@
++++
+linkTitle = "Thanos"
+title = "Thanos integration with prometheus"
+date = 2023-07-20T09:44:15+05:30
+weight = 20
++++
+
+This page explains how we can integrated [Thanos](https://thanos.io/) long term storage of metrics with KKP seed prometheus
+
+## Pre-requsites
+1. Helm is installed.
+1. KKP v2.22.4+ is installed in the cluster.
+1. KKP prometheus chart has been deployed in each seed where you want to store long term metrics
+
+## Integration steps
+Below page outlines
+1. Install thanos components in your kubernetes cluster via Helm chart
+1. Customize KKP prometheus chart to augment prometheus pod with thanos sidecar
+2. Customize KKP prometheus chart values to monitor and get alerts for Thanos components
```

Comment:
```suggestion
2. Customization of KKP Prometheus chart values to monitor get alerts for Thanos components
```

Reply:
I would say.. the `and` should be kept.. so my proposal would be
```
2. Customization of KKP Prometheus chart values to monitor and get alerts for Thanos components
```

Reply:
Check latest push.